R/raster-element.R

```diff
@@ -31,11 +31,14 @@
 #' @export
 #'
 #' @examples
-#' add_raster(file_name = "Rasterfiles.zip" ,
+#' add_raster(parent_element = list(),
+#'            file_name = "Rasterfiles.zip" ,
 #'            file_description = "A Raster File",
-#'            attribute_list =  add_attribute(attribute_name = "Date",
-#'                                            attribute_definition = "date data collected",
-#'                                            measurement_scale = "dateTime"),
+#'            attribute_list =  add_attribute(attribute_name = "Yrs", attribute_label = "Years", 
+#'                                            attribute_definition = "Calendar year of the observation from years 1990 - 2010.", 
+#'                                            storage_type = "integer", 
+#'                                            measurement_scale = "dateTime", date_time_format = "YYYY",
```

Comment:
in the example, please use the measurement scale and storage type data objects like in the other examples

Reply:
Fixed